modules/@apostrophecms/notification/index.js

```diff
@@ -141,26 +151,42 @@ module.exports = {
 
         Object.assign(notification, options);
 
-        return self.db.insertOne(notification);
+        return self.db.updateOne(
+          notification,
+          {
+            $set: notification,
+            $currentDate: {
+              updatedAt: true
+            }
+          }, {
+            upsert: true
+          });
       },
 
       // Resolves with an object with `notifications` and `dismissed`
       // properties.
       //
-      // If `options.displayingIds` is set, notifications
-      // whose `_id` properties appear in it are not returned.
+      // If `options.modifiedOnOrSince` is set, notifications
+      // greater than the timestamp are sent.
```

Comment:
notifications modified on or after that timestamp are sent, minus any notifications whose IDs are appended separated by commas after a colon `(:)`.

Reply:
Never mind, I like what you did by introducing `seenIds` better, please document that option.

Reply:
Ok, documented